entity-framework/core/modeling/generated-properties.md

```diff
@@ -47,48 +47,73 @@ Like `value generated on add`, if you specify a value for the property on a newl
 >
 > [!code-sql[Main](../../../samples/core/Modeling/FluentAPI/ValueGeneratedOnAddOrUpdate.sql)]
 
-## Conventions
+## Value generated on add
+
+By convention, non-composite primary keys of type short, int, long, or Guid are be set up to have values generated on add, and your database provider typically takes care of the necessary configuration. For example, an numeric primary key in SQL Server is automatically set up to be an IDENTITY column.
```

Comment:
It's weird to see "add" written like this. Consider "to have values generated for entities that will be inserted".

Reply:
Agree. Changed to the slightly shorter "to have values generated for inserted entities".